Userland/Libraries/LibJS/Runtime/AbstractOperations.cpp

```diff
@@ -47,6 +47,18 @@ Value require_object_coercible(GlobalObject& global_object, Value value)
     return value;
 }
 
+// 7.3.17 CreateArrayFromList ( elements ), https://tc39.es/ecma262/#sec-createarrayfromlist
+Value create_array_from_list(GlobalObject& global_object, MarkedValueList const& elements)
```

Comment:
This is already a thing, see `Array::create_from`

Reply:
:man_facepalming: I even searched for `CreateArrayFromList` in the code